pkgs/applications/audio/mpg123/default.nix

```diff
@@ -1,6 +1,6 @@
 { stdenv
 , fetchurl, alsaLib
-, buildPlatform, hostPlatform
+, hostPlatform
```

Comment:
If this is meant as an example of how stuff is done, perhaps put a reference to some URL or man page specifying what the symbol `hostPlatform` means, since many people won't know. You can argue that you want to keep Nix elitist or something like that.